.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,86 @@
+name: "Bug report 🐛"
+description: Report errors or unexpected behavior
+title: ''
```

Comment:
We could add here a prefix [BUG]

Reply:
We are using a different taxonomy for titles, adding [BUG] isn't helping.